docs/error-codes.md

```diff
@@ -39,6 +39,8 @@ Here is a summary of all the error codes thrown by the contract.
 | 121 | `fail_decision_callback` | Executing the proposal's decision callback results in failure. |
 | 123 | `unstake_invalid_proposal` | Cannot call `unstake_vote` on the proposal that is not flushed or dropped. |
 | 124 | `voter_does_not_exist` | The sender did not vote on the proposal or already unstaked tokens from the proposal. |
+| 125 | `proposal_handler_exists` | A proposal handler with the same name exists already |
+| 126 | `proposal_handler_not_found` | A proposal handler with the given name was not found |
```

Comment:
:thinking: should these be here?

AFAIK these are specific to the lambdaDAO variant (and the others based on it) whereas the error codes here are instead of baseDAO itself.

IOW shouldn't these be treated like `zero_mutez_err_msg` which is used by some variants but not all `baseDAO`-based contracts?

Reply:
Have removed these errors from the global list and made them to throw with `fail_proposal_check`.